server/src/main/java/org/elasticsearch/action/admin/cluster/node/tasks/cancel/TransportCancelTasksAction.java

```diff
@@ -276,4 +279,23 @@ public void messageReceived(final BanParentTaskRequest request, final TransportC
         }
     }
 
+    void cancelOrphanedTasks(Set<CancellableTask> orphanedTasks) {
+        if (orphanedTasks.isEmpty() == false) {
+            transportService.getThreadPool().generic().execute(new AbstractRunnable() {
+                @Override
+                public void onFailure(Exception e) {
+
```

Comment:
Maybe log a warning here?

Reply:
added in https://github.com/elastic/elasticsearch/pull/56620/commits/d4f7586c16d83ffea09b3f080e3a3d0a418ffeb3